DataFormats/L1TCorrelator/interface/TkEtMiss.h

```diff
@@ -0,0 +1,63 @@
+#ifndef TkTrigger_TkEtMiss_h
+#define TkTrigger_TkEtMiss_h
+// Package:     L1Trigger
+// Class  :     TkEtMiss
+// Original Author:  E. Perez
+//         Created:  Nov 14, 2013
+// Redesign Author: V. Rekovic
+//         Created: Mar 30, 2020
+
+// system include files
+// user include files
+#include "DataFormats/L1Trigger/interface/L1Candidate.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
+
+namespace l1t {
+  class TkEtMiss : public L1Candidate {
+  public:
+    enum EtMissType { kMET, kMHT, kNumTypes };
+    TkEtMiss();
+    TkEtMiss(const LorentzVector& p4,
+             EtMissType type,
+             const double& etTotal,
+             const double& etMissPU,
+             const double& etTotalPU,
+             const edm::Ref<TkPrimaryVertexCollection>& aVtxRef = edm::Ref<TkPrimaryVertexCollection>(),
+             int bx = 0);
+
+    TkEtMiss(const LorentzVector& p4,
+             EtMissType type,
+             const double& etTotal,
+             const double& etMissPU,
+             const double& etTotalPU,
+             int bx = 0);
+
+    // ---------- const member functions ---------------------
+    EtMissType type() const { return type_; }  // kMET or kMHT
+    // For type = kMET, this is |MET|; for type = kMHT, this is |MHT|
+    double etMiss() const { return et(); }
+    // For type = kMET, this is total ET; for type = kMHT, this is total HT
+    const double& etTotal() const { return etTot_; }
```

Comment:
I think you lost https://github.com/cms-sw/cmssw/pull/29346#discussion_r400587009